Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionPermission.h

```diff
@@ -75,6 +75,10 @@ WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionNativeMessagi
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
 WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionScripting;
 
+/*! @abstract The `sidePanel` permission requests access to the `chrome.sidePanel` APIs. */
```

Comment:
Even though this is a Chrome specific API currently, we should use `browser` to be neutral.

```suggestion
/*! @abstract The `sidePanel` permission requests access to the `browser.sidePanel` APIs. */
```